front/app/components/admin/UserBlockModals/messages.ts

```diff
@@ -8,11 +8,19 @@ export default defineMessages({
   subtitle: {
     id: 'app.containers.Admin.Users.userBlockModal.subtitle',
     defaultMessage:
-      'The selected user won’t be able to login to the platform for {daysBlocked}. If you wish to revert this, you can unblock them from the list of blocked users.',
+      "The selected user won't be able to log in to the platform for {daysBlocked}. If you wish to revert this, you can unblock them from the list of blocked users.",
   },
   daysBlocked: {
     id: 'app.containers.Admin.Users.userBlockModal.daysBlocked',
-    defaultMessage: '{numberOfDays} days',
+    defaultMessage: '{numberOfDays} {dayOrDays}',
+  },
+  day: {
+    id: 'app.containers.Admin.Users.userBlockModal.day',
+    defaultMessage: 'day',
+  },
+  days: {
+    id: 'app.containers.Admin.Users.userBlockModal.days',
+    defaultMessage: 'days',
```

Comment:
This is not how we should handle this one. Translators need to understand the full context, so we should rather do:

      '{numberOfDays, plural, one {1 day} other {{numberOfDays} days}}',

Reply:
ok thanks, thought this was the way but didn't find it on the docs

Reply:
👍 You can merge after making this change, no need for a new review round.